adoptopenjdk-api-v3-frontend/src/main/resources/META-INF/resources/index.html

```diff
@@ -27,6 +27,7 @@ <h2>Overview</h2>
 
     <h2>Usage</h2>
     <p>View usage at <a href="swagger-ui">swagger-ui docs</a></p>
+    For more information, including example queries, please look at <a rel="noopener noreferrer" href="https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/docs/STRUCTURE.md">STRUCTURE.md</a>
```

Comment:
should probably put that in a <p>

Reply:
@johnoliver what was the end of that sentence? :-)

Reply:
turns out github escapes `<p>`

Reply:
Oh yeah I'm dumb I dunno how I missed those